content/articles/introduction-to-udoo-x86-ii-ultra/index.md

```diff
@@ -0,0 +1,106 @@
+- [Introduction to Udoo X86 II Ultra](#introduction-to-udoo-x86-ii-ultra)
+- [Pros](#pros)
+- [Cons](#cons)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To follow through this article, you need to have a basic understanding of `session board controllers`
+
+### Introduction to Udoo X86 II Ultra
+Udoo X86 II Ultra session board controller's power is comparable to that of a new typical personal computer in the market today. Udoo X86 II Ultra comes embedded on the same platform with an Arduino Leonardo 101 board made with a more powerful X86 maker architecture.
+
+Udoo X86 II Ultra can run most of the applications that could be run on a PC.
+
+These applications include:
+- Professional development platforms
+- Graphical editors
+- 3D gaming software
+- Video streaming
+- Libraries
+- Sketches
+- The official Arduino Leonardo IDE
+- Applications present in the Arduino Leonardo space
+
+Udoo X86 II Ultra is a genuine computer. The absence of drivers or any related concern is not a problem. It supports different operating systems including, Windows 10, Android, and Linux.
+
+The embedded Arduino Leonardo 101 board comes with a six-axis accelerometer and a gyroscope built in. The customizability offered by the Arduino board makes this session board controller a very resourceful/powerful tool for casual uses.
+
+Udoo X86 II Ultra is based on the new generation Quad Core 64-bit personal computer with the intel's X86 processor architecture.
+
+With an energy consumption of about 5 to 6 Watts, the gigantic processor is concentrated in nanometers. The power can be provided via the VIN pins usually found on the board's head.
+
+### Pros
+Udoo X86 II Ultra session board has the following advantages:
+
+#### Enormous RAM
+Udoo X86 II Ultra comes with an 8GB built in RAM. The advanced version has 4GB of RAM while the basic one has 2GB of RAM. This makes the Udoo X86 II Ultra more powerful than most of the session board controllers on the market.
+
+#### Efficiency
+Udoo X86 II Ultra is capable of running software meant for a normal personal computer. This is made possible by a 2.56GHz quad-core intel CPU and an 8GB RAM.
+
+#### Availability of freely useful guides and community
+Through a helpful community even not all guides are available online.
+
+###	Lower energy consumptions
+The Arduino Leonardo 101 board requires less than 11 Watts of energy which makes it very energy efficient.
+
+#### Large storage capacity
+Udoo X86 II Ultra box comes with a 32GB embedded multimedia card. However, users might decide to upgrade the storage to fit their desire. Though expensive, an SSD card is usually the best option. Additionally, a micro-SD card can also be used as a storage media.
+
+#### X86-compatible architecture
+Udoo X86 II Ultra supports most operating systems including Linux, Windows and Android. For Linux and Android, the internal storage can be used as a read only memory with additional SATA `HDD/SSD/eMMC` drive used as /tmp and /home directories.
```

Comment:
I've noted a few acronyms that might be unfamiliar to beginners. Provide the full names and a short description of relevant abbreviations.

Reply:
Already Addressed.

Reply:
I can't see any updates yet! Did you forget to commit them?